interface-definitions/system_config-management.xml.in

```diff
@@ -67,6 +67,26 @@
               <constraintErrorMessage>Number of revisions must be between 0 and 65535</constraintErrorMessage>
             </properties>
           </leafNode>
+          <leafNode name="commit-confirm">
+            <properties>
+              <help>Commit confirm rollback type if no confirmation</help>
+              <completionHelp>
+                <list>reload reboot</list>
+              </completionHelp>
+              <valueHelp>
+                <format>reload</format>
+                <description>Reload previous configuration if not confirmed</description>
+              </valueHelp>
+              <valueHelp>
+                <format>reboot</format>
+                <description>Reboot to saved configuration if not confirmed</description>
+              </valueHelp>
+              <constraint>
+                <regex>(reload|reboot)</regex>
```

Comment:
Is it possible to use just “commit-confirm” without options (reload|restart) and use reload logic?
Even it won’t work perfect, it is ok for development branch

Reply:
Do you mean make a valueless node, say, 'commit-confirm-soft' (or a better name) with setting == reload ? Yes, that may be preferable if the name is clear ... I don't have strong feelings on the format; I do like having it a config option rather than an command line option to commit-confirm.

We should agree on the initial form, and after a bit of time in dev branch, we can confirm the syntax before we make available for Circinus.

Reply:
It may be easier to use the form with default value if we want to make global behavior changes similar to the rollback-soft/hard naming convention change:
https://vyos.dev/T5843

Reply:
If on the other hand, you meant drop any configuration and make soft revert the default behavior, that is a consideration; I'm considering what constraints are needed, and will update here before the conversation is resolved.

Reply:
We may want to leave the configuration node as valued for the possibility of adding further values, for example, 'reload-to-failsafe'.